src/cljs/webrtclojure/server-comms.cljs

```diff
@@ -17,6 +17,36 @@
 )
 
 ;;; -------------------------
+;;; Message handers for application routes
```

Comment:
I think the lines of code below should not be included in commit :octopus: 

20-49